agent/wpthook/test_state.cc

```diff
@@ -82,6 +82,7 @@ TestState::~TestState(void) {
 -----------------------------------------------------------------------------*/
 void TestState::Init() {
   Reset(false);
+  _timeout_start_time.QuadPart = 0;

```

Comment:
You should move it to the Reset() method. That's where the entire test_state is reset.